app/src/main/res/layout/fragment_login.xml

```diff
@@ -7,163 +7,182 @@
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:maxLength="255"
+                    android:imeOptions="actionNext"
+                    android:includeFontPadding="false"
+                    android:inputType="textEmailAddress"/>
+
+            </com.google.android.material.textfield.TextInputLayout>
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:id="@+id/forgotPassword"
+                android:text="@string/forgot_password"
+                android:layout_marginTop="@dimen/margin_padding_size_10"
+                app:layout_constraintStart_toStartOf="@id/tilPassword"
+                app:layout_constraintTop_toBottomOf="@id/tilPassword">
+
+            </TextView>
+
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/btnLogin"
+                android:layout_width="wrap_content"
+                android:layout_height="@dimen/standard_touch_size"
+                android:layout_marginTop="@dimen/margin_padding_size_10"
+                app:cornerRadius="20dp"
+                android:backgroundTint="@color/black"
+                android:insetTop="@dimen/margin_padding_size_0"
+                android:insetBottom="@dimen/margin_padding_size_0"
+                android:text="@string/btnLogin"
+                android:textSize="@dimen/text_size_14"
+                android:textStyle="bold"
+                app:icon="@drawable/ic_login_small"
+                app:iconTint="@null"
+                app:layout_constraintEnd_toEndOf="@+id/tilPassword"
+                app:layout_constraintTop_toBottomOf="@id/tilPassword" />
+
+<!--            <androidx.appcompat.widget.AppCompatSeekBar-->
+<!--                android:id="@+id/seekBar"-->
+<!--                android:layout_width="wrap_content"-->
+<!--                android:layout_height="wrap_content"-->
+<!--                android:layout_marginHorizontal="@dimen/margin_padding_size_46"-->
+<!--                android:layout_marginTop="@dimen/margin_padding_size_10"-->
+<!--                app:layout_constraintStart_toStartOf="@+id/forgotPassword"-->
+<!--                app:layout_constraintTop_toBottomOf="@+id/forgotPassword" />-->
+
+            <TextView
+                android:id="@+id/signUp"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_padding_size_34"
+                android:text="@string/signUp"
+                android:textColor="@color/black"
+                android:textSize="@dimen/text_size_18"
+                android:textStyle="bold"
+                app:layout_constraintStart_toStartOf="@id/forgotPassword"
+                app:layout_constraintTop_toBottomOf="@id/tilPassword" />
```

Comment:
Don't add buttons without functionality. Normally the GUI is part of the feature. So don't do it, when you are not working on the feature. For now: choose a design (textView vs seekbar) and leave it in commented or delete it completely.

Reply:
removed both